pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -328,6 +328,13 @@ func (plugin *cniNetworkPlugin) TearDownPod(namespace string, name string, id ku
 		klog.Warningf("CNI failed to retrieve network namespace path: %v", err)
 	}
 
+	// Windows doesn't have loNetwork. It comes only with Linux
+	if plugin.loNetwork != nil {
+		if err = plugin.deleteFromNetwork(plugin.loNetwork, name, namespace, id, netnsPath, nil); err != nil {
+			return err
```

Comment:
Currently, if we fail to delete the `loNetwork`, we won't attempt to delete the `getDefaultNetwork`. Is that expected?

Or should we aggregate errors and return them all at the end?

Reply:
@mars1024 I would say just ignore the failure. the loopback network doesn't actually need any cleanup, so we can ignore errors here and they shouldn't be fatal.

Reply:
Thanks @dcbw  @mattjmcnaughton , will update this!